app/form_models/publishers/job_listing/contract_type_form.rb

```diff
@@ -1,11 +1,11 @@
 class Publishers::JobListing::ContractTypeForm < Publishers::JobListing::VacancyForm
-  attr_accessor :contract_type, :fixed_term_contract_duration, :parental_leave_cover_contract_duration
+  attr_accessor :contract_type, :fixed_term_contract_duration, :parental_leave_cover_contract_duration, :is_parental_leave_cover
```

Comment:
Question: As we removed the field from the view, shouldn't we also remove all `parental_leave_cover_contract_duration` references from this form?